stack/applyingstackpolicy.go

```diff
@@ -0,0 +1,45 @@
+package stack
+
+import (
+	"github.com/Appliscale/perun/context"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+// ApplyStackPolicy creates StackPolicy from JSON file.
+func ApplyStackPolicy(context *context.Context) {
+	template, stackName, incorrectPath := getTemplateFromFile(context)
+	if incorrectPath != nil {
+		context.Logger.Error(incorrectPath.Error())
+		return
```

Comment:
If you are creating a function that can be used outside package always return error.